sherpa-onnx/csrc/onnx-utils.h

```diff
@@ -89,6 +90,21 @@ std::vector<char> ReadFile(AAssetManager *mgr, const std::string &filename);
 // TODO(fangjun): Document it
 Ort::Value Repeat(OrtAllocator *allocator, Ort::Value *cur_encoder_out,
                   const std::vector<int32_t> &hyps_num_split);
+
+struct CopyableOrtValue {
+  Ort::Value value{nullptr};
+
+  CopyableOrtValue() {}
```

Comment:
We can add another constructor
```cpp
/* explicit */ CopyableOrtValue(Ort::Value v) // NOLINT
  : value(std::move(v)) {}
```

We make it implicit so that you can use
`std::move` later to move a vector of OrtValue to a vector of CopyableOrtValue.